db/write_batch.cc

```diff
@@ -1586,7 +1593,7 @@ class MemTableInserter : public WriteBatch::Handler {
     }
   }
 
-  Status MarkBeginPrepare() override {
+  Status MarkBeginPrepare(bool unprepare) override {
```

Comment:
since the term "unprepare" is counter-intuitive here, can you add an inline comment to explain it again.